### PR TITLE
CON-1054 - Moved hello-world sample from SDK repo

### PR DESCRIPTION
**Pull request by filipesoliveira**
The hello-world sample was moved from conclave-sdk.

### CODE REVIEW

**Review by ac101m — COMMENTED** [review]
Looks good to me. Will leave it to @shamsasari to approve though as he wanted to take a look.

File: README.md [hunk 1]
```diff
@@ -37,6 +37,9 @@ This sample allows bidders to confidentially submit their bids to an enclave (A
 ### [Conclave Corda Trade](./conclave-corda-trade):
 This application serves as a demo for building a confidential trading system based on Corda and Conclave. An exchange Corda node would serve as a host which runs the enclave, while broker nodes serve as clients which send encrypted orders from their end-clients which are matched in the enclave and trades generated are recorded in all relevant participants ledgers.
 
+### [Conclave Hello World](./hello-world):
+This is a simple app using the Conclave API. It is licensed under the Apache 2 license, and therefore you may copy/paste it to act as the basis of your own commercial or open source apps.
```
Comment (shamsasari) [inline README.md]: This is a simple app using the Conclave API and functions as a tutorial.

File: hello-world/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Conclave Hello World
+
+This is a simple app using the Conclave API. It is licensed under the Apache 2 license, and therefore you may 
+copy/paste it to act as the basis of your own commercial or open source apps.
+
+## How to run
+
+Start the host on a Linux system, which will build the enclave and host:
```
Comment (shamsasari) [inline hello-world/README.md]: You can actually build this on non-Linux systems, as described in the docsite. In fact there is no need to duplicate this information from the docsite. This sample is the docsite's tutorial, just have this README reference the correct page on the docsite.

File: hello-world/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+```bash
+./gradlew client:shadowJar
+java -jar client/build/libs/client.jar "S:0000000000000000000000000000000000000000000000000000000000000000 PROD:1 SEC:INSECURE" "reverse me"
+```
+
+## Next step
+
+To understand what the first string parameter to the client is or why the host jar has "mock" in the file name then 
+head over to our [online docsite](https://docs.conclave.net/running-hello-world.html). A local copy is also 
+available in the SDK zip.
```
Comment (shamsasari) [inline hello-world/README.md]: Not correct anymore

File: README.md [hunk 4]
```diff
@@ -37,6 +37,9 @@ This sample allows bidders to confidentially submit their bids to an enclave (A
 ### [Conclave Corda Trade](./conclave-corda-trade):
 This application serves as a demo for building a confidential trading system based on Corda and Conclave. An exchange Corda node would serve as a host which runs the enclave, while broker nodes serve as clients which send encrypted orders from their end-clients which are matched in the enclave and trades generated are recorded in all relevant participants ledgers.
 
+### [Conclave Hello World](./hello-world):
+This is a simple app using the Conclave API and functions as a tutorial.
```
Comment (shamsasari) [inline README.md]: Sorry, my comment was just for the first setenence. The rest of the paragraph was fine.

File: hello-world/gradle.properties [hunk 5]
```diff
@@ -0,0 +1,3 @@
+#
```
Comment (shamsasari) [inline hello-world/gradle.properties]: ?

**Review by shamsasari — APPROVED** [review]

**Pull request merged**